android/src/main/java/com/bitmovin/player/reactnative/RNPlayerViewManager.kt

```diff
@@ -217,13 +224,18 @@ class RNPlayerViewManager(private val context: ReactApplicationContext) : Simple
     private fun attachPlayer(view: RNPlayerView, playerId: NativeId?, playerConfig: ReadableMap?) {
         Handler(Looper.getMainLooper()).post {
             val player = getPlayerModule()?.getPlayer(playerId)
-            playerConfig
-                ?.getMap("playbackConfig")
-                ?.getBooleanOrNull("isPictureInPictureEnabled")
-                ?.let {
-                    pictureInPictureHandler.isPictureInPictureEnabled = it
-                    view.pictureInPictureHandler = pictureInPictureHandler
-                }
+            val isPictureInPictureEnabled = (view.pictureInPictureConfig?.isEnabled ?: false)
+                .or(
+                    playerConfig
+                        ?.getMap("playbackConfig")
+                        ?.getBooleanOrNull("isPictureInPictureEnabled")
+                        ?: false,
+                )
+
+            isPictureInPictureEnabled.let {
+                pictureInPictureHandler.isPictureInPictureEnabled = it
+                view.pictureInPictureHandler = pictureInPictureHandler
+            }
```

Comment:
1.) The `or()` function is discouraged not only because of style but also because both sides of `or` will always be evaluated as opposed to E.g. `somethingTrue()` || `somethingFalse()` that will only evaluate the left side if true.
2.) As mentioned above I think we can and should remove the pictureInPictureHandler in this file
3.) `isPictureInPictureEnabled.let` only makes the code confusing in this case

```suggestion
            val playbackConfig = playerConfig?.getMap("playbackConfig")
            val isPictureInPictureEnabled = view.pictureInPictureConfig?.isEnabled == true ||
                    playbackConfig?.getBooleanOrNull("isPictureInPictureEnabled") == true
            view.pictureInPictureHandler?.isPictureInPictureEnabled = isPictureInPictureEnabled
```

Reply:
I literally expected feedback on this one (maybe should have even explicitly ask for it) as I'm no expert of Kotlin code style.

Thank you for the excellent and simple suggestion!
Implemented in 0b2460f

Reply:
Thanks for catching!
It looks like my former refactoring in 7872455ce1c9b67927d992e70dda6c5765af3a7a and the suggested changes didn't play well together.

Fixed in ae24ef2